src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="GitHubActionsTestLogger" Version="2.3.3" PrivateAssets="All" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0-release-23468-02" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="3.0.0" />
```

Comment:
Core should not have any references to downstream packages, so you can't add this package reference.

Reply:
@bording  Even to test projects?

Reply:
Yes, even test projects. The AcceptanceTests project produces a source package that is used downsteram.

Even for test projects in Core that don't produce packages, adding a downstream package adds a transitive reference to the Core package, which means you now have a Core project referencing itself.

Reply:
Removed the reference and hence the test.